database/migrations.php

```diff
@@ -5,7 +5,7 @@
  * @param mysqli $db
  * @return void
  */
-function migate(mysqli $db)
+function migrate(mysqli $db)
```

Comment:
This is more of a general comment, but if you fix  a typo (which you should, thanks!) make sure you update wherever the function/variable is used to avoid errors)

Reply:
Ok, had a look, found where the function was called (found one place where the function was called)